frontend/views/components/Search.vue

```diff
@@ -0,0 +1,83 @@
+<template lang='pug'>
+form.c-search-form(
+  @submit.prevent='submit'
+)
+  label.field
+    .sr-only {{label}}
+
+    .input-combo
+      .is-icon(:aria-label='L("Search")')
+        i.icon-search
+
+      input.input(
+        type='text'
+        name='search'
+        data-test='search'
+        :placeholder='placeholder'
+        :value='value'
+        @input='$emit("input", $event.target.value)'
+      )
+
+      button.is-icon-small(
+        v-if='value !== ""'
+        :aria-label='L("Clear search")'
+        @click='$emit("input", "")'
+      )
+        i.icon-times
+</template>
+
+<script>
+import L from '@view-utils/translations.js'
+
+export default {
+  name: 'Search',
+
+  props: {
+    value: {
+      type: String,
+      required: true
+    },
+
+    placeholder: {
+      type: String,
+      default: L('Search...')
+    },
+
+    label: {
+      type: String,
+      required: true
+    }
+  },
+
+  watch: {
+    value () {
+      this.$emit('input', this.value)
```

Comment:
maybe `emit` should be called `change` to follow the native event name `@change`.

Reply:
Oh, nevermind. I got it now. by using `input`, the `v-model` bind happens automatically!

Reply:
Correct, it was a test i forgot to replace by shorthand